### ISSUE TITLE
German deck - some words need to have an article (die / der) in their name

### ISSUE THREAD

**Issue by avshyz** [anki-geo/ultimate-geography#543]
As the title states, some names of the countries are inaccurate because they omit the article (die / der).
Here's the full list:

der Iran
der Irak
der Jemen
der Tschad
der Vatikan
die Dominikanische Republik
die Mongolei
die Schweiz
die Slowakei
die Ukraine
die Türkei

die Bahamas
die Färöer
die Niederlande
die Philippinen
die Vereinigten Arabischen Emirate
die Vereinigten Staaten von Amerika

**Comment by aplaice** [comment]
Omitting the articles is [our current policy](https://github.com/anki-geo/ultimate-geography/blob/master/CONTRIBUTING.md#country-field) ([see also directly here](https://github.com/anki-geo/ultimate-geography/pull/157#issuecomment-549143860)).

In brief, the motivation is that if Wikipedia omits the grammatical article in the article title corresponding to the country, then it means that the article isn't actually part of the name of the country itself.

Comparing with English, I don't think that omitting, say, the "die" from before "Vereinigten Staaten von Amerika" in German, is any different than omitting the "the" from before "United States of America" in English — in both cases, if one uses the name in a sentence, one includes the article, but if it's just used in isolation (e.g. as a title, or in a list — for instance [see here](https://de.wikipedia.org/wiki/Europ%C3%A4ische_Union#Mitgliedstaaten) ("Slowakei")) one omits it.

<hr/>

Obviously, we might be wrong (this is a tricky topic and convention varies between languages), so please do feel free to argue in favour of including articles!

**Comment by avshyz** [comment]
Firstly—thank you for your quick reply.
I must admit that I thought it's a German thing (to add articles to land-names), so when I browsed through the issue list I've looked for German related ones (which is why I didn't find the previous discussions about it). I'd also admit that I'm still learning German, which is why I have a personal stake in this matter :).

Anyway, you make a compelling argument (I wasn't aware the article is omitted when the name's being used in isolation).
